packages/fx-core/tests/plugins/solution/solution.usertask.test.ts

```diff
@@ -283,7 +283,8 @@ describe("V2 implementation", () => {
     mockScaffoldCodeThatAlwaysSucceeds(appStudioPluginV2);
     mockScaffoldCodeThatAlwaysSucceeds(localDebugPluginV2);
     mockScaffoldCodeThatAlwaysSucceeds(frontendPluginV2);
-
+    const insiderPreviewFlag = process.env.TEAMSFX_INSIDER_PREVIEW;
```

Comment:
@1yefuwang1 seems there need to sync up with you, on "v2/executeUserTask" generateArmTemplate need to support selectedPlugins parameters